pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -233,39 +234,66 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 	}, nil
 }
 
+// The following function expects the uniqueName to follow the fromat plugin_name/volume_name and the plugin
+// name must be namespaced as descibed by the plugin interface, i.e. namespace/plugin containing exactly one '/'.
+// This means the unique name will always be in the form of plugin_namespace/plugin/volume_name
+// see k8s.io/kubernetes/pkg/volume/plugins.go VolumePlugin interface description and
+// pkg/volume/util/volumehelper/volumehelper.go GetUniqueVolumeNameFromSpec that constructs the unique volume names.
+func (og *operationGenerator) splitUniqueName(uniqueName v1.UniqueVolumeName) (string, string, error) {
```

Comment:
OK. Will move it to the volumehelper where it was in the original patch version.